packages/app/src/components/DebuggerConnectPanel.tsx

```diff
@@ -73,10 +74,16 @@ export type DebuggerConnectPanelProps = {
 };
 
 export const DebuggerConnectPanel: FC<DebuggerConnectPanelProps> = ({ onConnect, onCancel }) => {
-  const [connectUrl, setConnectUrl] = useState('ws://localhost:21888');
+  const [defaultConnectUrl, setDefaultConnectUrl] = useRecoilState(debuggerDefaultUrlState);
+  const [connectUrl, setConnectUrl] = useState(defaultConnectUrl);
 
   const textField = useRef<HTMLInputElement>(null);
 
+  function doConnect() {
+    onConnect?.(connectUrl);
+    setDefaultConnectUrl(connectUrl);
+  }
+
```

Comment:
This would be an edge case for using `useSafeCallback()` 😄

Reply:
Yeah, I'm not too worried about rerenders for this panel